estate/models/buildings_model.py

```diff
@@ -0,0 +1,17 @@
+from odoo import models, fields   # noqa: F401
+
+class buidlings_model(models.Model):
+    _name = 'estate.buildings'
+    _description = 'Buildings Model'
```

Comment:
Please use double quotes for non-technical strings
```suggestion
    _description = "Buildings Model"
```